pkg/manager/manager_bgp.go

```diff
@@ -0,0 +1,118 @@
+package manager
+
+import (
+	"context"
+	"os"
+	"syscall"
+
+	"github.com/packethost/packngo"
+	"github.com/plunder-app/kube-vip/pkg/bgp"
+	"github.com/plunder-app/kube-vip/pkg/cluster"
+	"github.com/plunder-app/kube-vip/pkg/packet"
+	log "github.com/sirupsen/logrus"
+)
+
+// Start will begin the Manager, which will start services and watch the configmap
+func (sm *Manager) startBGP() error {
+	var cpCluster *cluster.Cluster
+	//var ns string
+	var err error
+
+	// If Packet is enabled then we can begin our preperation work
+	var packetClient *packngo.Client
+	if sm.config.EnablePacket {
+		var projectID string
+		if sm.config.ProviderConfig != "" {
+			key, project, err := packet.GetPacketConfig(sm.config.ProviderConfig)
+			if err != nil {
+				log.Error(err)
+			} else {
+				// Set the environment variable with the key for the project
+				os.Setenv("PACKET_AUTH_TOKEN", key)
+				// Update the configuration with the project key
+				projectID = project
+			}
+		}
+		packetClient, err = packngo.NewClient()
```

Comment:
It would be nice to include the UserAgent and a ConsumerID here. (ConsumerID can be string, but it doesn't vary between versions like UserAgent)

https://github.com/packethost/packet-cli/blob/2759d28b097c0f735e7324ee40a86a175d3ce71b/cmd/root.go#L60-L64